addons/cookoff/XEH_postInit.sqf

```diff
@@ -62,6 +62,9 @@ if (hasInterface) then {
         params ["_object", "_sound"];
 
         if (isNull _object) exitWith {};
+        
+        // If the sound is disabled, we exit.
+        if(!GVAR(cookoffDisableSound)) exitWith {};
```

Comment:
```suggestion
        if (!GVAR(cookoffDisableSound)) exitWith {};
```

Also, move to before line 62 - filter it out immediately instead of waiting if the setting is disabled.

Reply:
If you are making the setting to be global for everyone, then filter out the global events for it. It's a waste to send an event to everyone, just for everyone to locally decide that they should essentially discard it.

So either filter it out before or make the CBA setting a `0` setting, so that everyone can change it individually.

Reply:
> If you are making the setting to be global for everyone, then filter out the global events for it. It's a waste to send an event to everyone, just for everyone to locally decide that they should essentially discard it.
> 
> So either filter it out before or make the CBA setting a `0` setting, so that everyone can change it individually.

This hasn't been resolved.